modules/ROOT/pages/jquery.adoc

```diff
@@ -0,0 +1,125 @@
+= TinyMCE jQuery integration quick start guide
+
+:title_nav: jQuery
+:description: Documentation for the official TinyMCE jQuery integration.
+:keywords: integration integrate jquery javascript
+:productSource: cloud
+
+The https://github.com/tinymce/tinymce-jquery[Official {productname} jQuery component] integrates {productname} into jQuery projects.
+
+
+NOTE: {companyname} is pleased to provide integrations/code guidance to help you build great products with {productname}. If you have queries about this integration, post your question on Stack Overflow using the link:{communitysupporturl}[`{prodnamecode}` tag].
+
+== Procedure
+This procedure creates a basic jQuery integration containing a {productname} editor based on our xref:basic-example.adoc[Basic example].
+
+. In a HTML file, add a source script for source jQuery.
++
+If the project loads jQuery from https://code.jquery.com/, use the script provided by the https://code.jquery.com/[jQuery CDN], which includes the `integrity` and `crossorigin` attributes.
+
+. Source {productname} and the {productname} jQuery integration from the {cloudname} or from a self-hosted location.
+
+* *{cloudname}*
++
+To source {productname} from the {cloudname} and the {productname} jQuery integration from a CDN, add the following `script` elements:
++
+[source,html,subs="attributes+"]
+----
+<script src="{cdnurl}" referrerpolicy="origin"></script>
+<script src="https://cdn.jsdelivr.net/npm/@tinymce/tinymce-jquery@1/dist/tinymce-jquery.min.js"></script>
+----
++
+Replace `no-api-key` in the source script (`<script src=...`) with a {cloudname} API key, which is created when signing up to the link:{accountsignup}/[{cloudname}].
++
+Signing up for a {cloudname} API key will also provide a trial of the link:premium.html[Premium Plugins].
+
+* *{productname} self-hosted*
++
+To use an independent deployment of {productname} and the jQuery integration, add source scripts to either the `<head>` or the end of the `<body>` of the HTML file, such as:
++
+[source,html]
+----
+<script src="/public/tinymce/tinymce.min.js"></script>
+<script src="/public/tinymce-jquery/tinymce-jquery.min.js"></script>
+----
++
+The files will normally be sourced from npm:
++
+[source,shell]
+----
+npm install --save tinymce @tinymce/tinymce-jquery
+----
+and then copied to the public directory of your webserver by the compilation process.
++
+For information on self-hosting {productname}, see: link:advanced-install.html[Installing {productname}].
```

Comment:
I don't know what to replace this link with. Any suggestions?

- [ ] Replace link